tests/kafkatest/sanity_checks/test_console_consumer.py

```diff
@@ -44,8 +44,7 @@ def setUp(self):
     @cluster(num_nodes=3)
     @matrix(security_protocol=['PLAINTEXT', 'SSL'], metadata_quorum=quorum.all_kraft)
     @cluster(num_nodes=4)
-    @matrix(security_protocol=['SASL_SSL'], sasl_mechanism=['PLAIN'], metadata_quorum=quorum.all_kraft)
-    @matrix(security_protocol=['SASL_SSL'], sasl_mechanism=['SCRAM-SHA-256', 'SCRAM-SHA-512']) # SCRAM not yet supported with KRaft
+    @matrix(security_protocol=['SASL_SSL'], sasl_mechanism=['PLAIN', 'SCRAM-SHA-256', 'SCRAM-SHA-512'], metadata_quorum=quorum.all_kraft)
```

Comment:
While we are here, I wonder if we could directly remove the usage of `ZookeeperService` in this class. What do you think?

Reply:
Sure, I'll give it a try.

Reply:
I have just removed the `ZookeeperService` and run the tests on my local. Here is the result:
![Screenshot from 2025-01-31 17-55-15](https://github.com/user-attachments/assets/d0cdaf68-ab81-4e60-b5a6-ceeadd94c364)